extensions/prototype-content-with-issues-table/app.R

```diff
@@ -69,37 +99,29 @@ server <- function(input, output, session) {
   client <- connect()
   
   # TODO: use `v1/content/failed` when #30414 merges so we only list content we
-  # know has failed before, filter to deployed within last 60d for now
+  # know has failed before, filter to deployed within last year for now
```

Comment:
It landed as an unversioned endpoint now 🙌 , I'm eager to see how much a difference does it make